test/AsyncAPI.Saunter.Generator.Cli.Tests/DotnetCliToolTests.cs

```diff
@@ -0,0 +1,234 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
in fact, these are e2e tests, they will be extremely difficult to maintain, I suggest not trying to go this way and start with classic unit tests